flair/embeddings/transformer.py

```diff
@@ -809,11 +809,26 @@ def __init__(
             # load tokenizer from inmemory zip-file
             self.tokenizer = self._tokenizer_from_bytes(tokenizer_data)
 
+        def is_supported_t5_model(config: PretrainedConfig) -> bool:
+            t5_supported_model_types = ["t5", "mt5", "longt5"]
+            return getattr(config, "model_type", "") in t5_supported_model_types
```

Comment:
I wasn't able to find a model that has no `model_type` set, so maybe this could be shortened to `return config.model_type in t5_supported_model_types`.